client/components/claim/claim/ClaimOgv.tsx

```diff
@@ -55,6 +47,16 @@ const ClaimOgv: FunctionComponent<ClaimOgvProps> = () => {
     loading: maxOgvLockupRewardLoading,
   } = useStakingAPY(claimableOgv, 48);
 
+  if (!claim.loaded || !claim.optional.hasClaim) {
+    return <></>;
+  }
+
+  // as specified here: https://github.com/OriginProtocol/ousd-governance/blob/master/contracts/OgvStaking.sol#L21
+  const votingDecayFactor = 1.8;
+
+  const veOgvFromOgvLockup =
+    claimableOgv * votingDecayFactor ** (lockupDuration / 12);
```

Comment:
We've been using similar formulas that have been replaced when calculating the APY. Curious are these still correct?

Reply:
That's a good question, claim page isn't useable by anyone right now (since the Claim window has long passed) and all unclaimed OGV have been burned. So I wasn't paying much attention to this page/component as it's never rendered 😅 

Made this change to fix the prod deployment. Could delete this file but I'm not too sure about that

Reply:
ah yes you are right it is not important... no use spending more time on this imho. LGTM